Cangjie5.txt

```diff
@@ -24455,7 +24455,6 @@
 𮖶	hcyhv
 𪴿	hcylm
 𰙺	hcylm
-𥬧	hcym
```

Comment:
The mapping is correct and it should not be changed.

![image](https://user-images.githubusercontent.com/3607926/133942537-a5edea3d-4d7f-4200-a93f-93f9b60fe631.png)